src/bitmap.c

```diff
@@ -1024,21 +1024,10 @@ GdipCreateBitmapFromHICON (HICON hicon, GpBitmap** bitmap)
 
 	status = GdipCloneImage ((GpImage *)hicon, (GpImage**)bitmap);
 	if (status == Ok) {
-		/* ColorPalette definition includes one ARGB member (ARGB Entries[1]). In reality there are 
-		 * Count entries allocated, where Count can be 0 (and required to substract the ARGB size)
-		 */
-		/* coverity[buffer_alloc] */
-		ColorPalette *palette = (ColorPalette*)GdipAlloc (sizeof (ColorPalette) - sizeof (ARGB));
-		if (!palette)
-			return OutOfMemory;
-
-		palette->Flags = 0;
-		palette->Count = 0;
-		status = GdipSetImagePalette ((GpImage*)*bitmap, palette);
+		(*bitmap)->active_bitmap>palette = NULL;
```

Comment:
missing the dash in `->`.

also should we free the old `palette` before setting it to NULL?